### PR TITLE
Bug 1157128 - Remove broken l10n ids

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9f9533dccf5fe4d88e51c62fcc59dcac081879c5

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c96afea62295e97a229ddf633f733fab1cd917b5

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2786a409c0902a20bffc2492e1270307be2e268f

File: apps/system/test/marionette/app_authentication_dialog_test.js [hunk 1]
```diff
@@ -76,4 +76,28 @@ marionette('App Authentication Dialog',
       return !authDialog.displayed();
     });
   });
+
+  test('user can login', function() {
```
Comment (KevinGrandon) [inline apps/system/test/marionette/app_authentication_dialog_test.js]: I wonder if this should just be squashed into the previous test-case? Seems it's better to actually login then to just unprotect the URL?

**Pull request merged**